src/openfluid/utilsq/RESTClient.cpp

```diff
@@ -155,13 +155,15 @@ RESTClient::RESTClient()
 
 void RESTClient::removeRawHeader(const QByteArray Key)
 {
+  t_HeadersList NewHeaders;
```

Comment:
Why these modifications?

Reply:
I assume it was to avoid changing the list i am iterating on, but not sure anymore if this was really necessary.